### PR TITLE
Prevent top-right map controls from going out of screen when Attribute Table is open

### LINKED ISSUE #488: Rightmost map controls gets cropped when Attribute Table is open
### Checklist

- [x] I've searched through the existing [issues](https://github.com/g3w-suite/g3w-client/issues) and this bug has never been reported before

## Screen size: 1536 x 768 px

image

### Steps to reproduce

1. right-click on a layer in the legend
2. click on **"Open Attribute table"** context menu item
3. rightmost map controls are cut off the screen

### Environment

[g3wsdk.info]

- g3w-admin: __3.6.dev-20230920133849__
- g3w-client: __3.9.0-alpha.1__
- browser: __Firefox 117.0__
- operating system: __Windows 11 64-bit__

### Link to your project

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by volterra79**
Closes: #488 

Screenshot from 2023-10-11 16-18-24

### CODE REVIEW

File: src/app/gui/map/mapservice.js [hunk 1]
```diff
@@ -1341,9 +1341,12 @@ proto._updateMapControlsLayout = function({width, height}={}) {
...
         if (changedAndMoreSpace.changed) {
           const mapControslHeight = this.state.mapControl.grid[this.state.mapControl.currentIndex].columns * this.state.mapcontrolSizes.minWidth;
-          const mapControlsWidth = this.state.mapControl.grid[this.state.mapControl.currentIndex].rows * this.state.mapcontrolSizes.minWidth;
           this.state.mapcontrolDOM.css('height', `${mapControslHeight}px`);
-          this.state.mapcontrolDOM.css('width', `${mapControlsWidth}px`);
+          /**
+           *  @deprecate since v3.9.0.
+           */
+          //const mapControlsWidth = this.state.mapControl.grid[this.state.mapControl.currentIndex].rows * this.state.mapcontrolSizes.minWidth;
+          //this.state.mapcontrolDOM.css('width', `${mapControlsWidth}px`);
```
Comment (Raruto) [inline src/app/gui/map/mapservice.js]: @volterra79 not so easy to understand, what has been deprecated?

Comment (volterra79) [reply src/app/gui/map/mapservice.js]: @Raruto because display:flex property do it automatically

**Comment by Raruto** [comment]
Nope.. 🤔

image

 [g3wsdk.info]

- g3w-admin: __3.6.update_one_to_one_join_info-20231005124956__
- g3w-client: __3.9.0-alpha.1__
- browser: __Firefox 118.0__
- operating system: __Windows 10 64-bit__

**Comment by Raruto** [comment]
now it's better, although map buttons now overlap (and tooltips are below) the expanded attribute table:

image

**Review by Raruto — APPROVED** [review]

**Review by Raruto — APPROVED** [review]

**Pull request merged**